dev/Breadcrumb/BreadcrumbBar_themeresources.xaml

```diff
@@ -15,6 +16,7 @@
     <Thickness x:Key="BreadcrumbBarItemDefaultMargin">0,3,0,4</Thickness>
     <Thickness x:Key="BreadcrumbBarItemLastItemMargin">0,3,1,4</Thickness>
```

Comment:
Can you move these into themeresources (duplicate for ligth/highcontrast) and verify that you can customize this from the app ?